<launch>
    <arg name="low_level_Config" default="$(find dawg_core)/config/low_level_Config.yaml" />
    <rosparam file="$(arg low_level_Config)" command="load" />
    <node pkg="dawg_core" type="dawg_hl_control.py" name="high_level_controller" output="screen"/>
    <node pkg="dawg_core" type="dawg_ll_control" name="low_level_controller" output="screen"/>
</launch>